### DISCUSSION TITLE
ItemStack.displayName to .name for standardization

### DISCUSSION THREAD

**Discussion by Sombian** [Minestom/Minestom#1877] [General]
both `Lore` and `Name` are under `ItemStack`'s `display` tag.
but `Lore` can be modified with method `.lore` whereas `Name` is `.displayName`.
i think its better to rename them.

**Comment by emortaldev** [comment]
Should probably be an issue or even better just debated on the Discord, however this would just create unnecessary problems when I'm pretty sure this is also how Spigot does things
